cli/command/streams.go

```diff
@@ -0,0 +1,23 @@
+package command
+
+import (
+	"github.com/docker/cli/cli/streams"
+)
+
+// InStream is an input stream used by the DockerCli to read user input
+// Deprecated: Use github.com/docker/cli/cli/streams.In instead
```

Comment:
When do we expect we would be able to get rid of these if not right now?

Reply:
I don't think we have an official depreciation plan, but I think that 2 releases is a good start, so let's say we will remove them after the 19.09 release? Do you want me to add a comment about it in the code?